# Exam 2

Just generic python setup first, including some helper functions to help with degree-radians conversion, and solving quadratic equations

In [1]:
import numpy as np
from numpy import sin, cos, tan, arctan2, sqrt

In [2]:
def rad(th):
    return th * np.pi/180

def deg(th):
    return th * 180/np.pi

def quadratic(a, b, c):
    return ((-b + sqrt(b**2 - 4*a*c)) / (2*a),
            (-b - sqrt(b**2 - 4*a*c)) / (2*a))

## Problem 1

* (a) equal (because $a = 0$)
* (b) equal (third law pair)
* (c) less than (because $a < 0$, ie, downward)
* (d) equal (third law pair)
* (e) unknown (because only $v$ is known, but not $a$. E.g., down at constant speed would meean $a = 0$, so forces are equal, but it could also be that it's going downward while speeding up or slowing down.
* (f) equal (third law pair)







## Problem 2
### Part a)

Gravity and normal force act on the passenger. At the top of the loop, they both point straight down, so I'll make that my positive direction.

[One should draw a FBD]

Acceleration is centripetal acceleration because the roller coaster is moving on a circular track (and at that point neither slowing down nor speeding up).

$$
m\frac{v^2}{R} = ma = F_{net} = mg + F_N
$$

From this equation one can tell that the larger the speed, the larger the normal force has to be (makes sense, getting pushed into the seat), since $mg$ doesn't change. However, as $v$ becomes slower and slower, $F_N$ would have to become negative, which it cannot. (A reasonable passenger, however, would probably try to hold on to the roller coaster car and might be able to pull himself up (negative force given my choice of direction)). But if you don't do this, $F_N = 0$ is where you become weightless and just about don't fall out.

$$
m\frac{v^2}{R} = mg \Longrightarrow
v = \sqrt{Rg}
$$

Let's put in the numbers:

In [3]:
R = 10
g = 9.8
print("v = {:f} m/s".format(sqrt(R*g)))

v = 9.899495 m/s


### Part b)

Maybe one should draw a FBD here, too. Though it's not really necessary.

$$a = a_c = \frac{v^2}{R}$$

Putting in numbers:

In [4]:
v_bottom = 22.1
a_bottom = v_bottom**2 / R
print("a = {:f} m/s^2 = {:f} g".format(a_bottom, a_bottom/g))

a = 48.841000 m/s^2 = 4.983776 g


At $5g$, that ride should be fun...

## Problem 3

[Draw a FBD]. We have gravity, normal force and the friction force of the tires on the road, which is what's accelerating the car.

Use $x$ and $y$ coordinates as usual ($y$ being vertical, positive up). From Newton's 2nd Law in the $y$ direction:

$$
0 = ma_y = F_{net,y} = F_N - mg \Longrightarrow F_N = mg
$$

In the x-direction:
$$
m a = m a_x = F_{net,x} = F_{f,s} \le \mu_s F_N \Longrightarrow
a \le \mu_s g
$$

Under ideal condition (good tires, dry), the maximum $\mu_s$ from the table is 0.8. So the maximum acceleration can be calculated

In [5]:
mu_s = 0.8
a_max = mu_s*g
print("a_max = {:f} m/s^2".format(a_max))

a_max = 7.840000 m/s^2


A car accelerating at $a_{max}$ for a distance of $100$ ft from a step would be going
$$
v_f^2 - v_i^2 = 2 a_{max} d \Longrightarrow v_f = \sqrt{2a_{max}d}
$$
Let's calculate that and convert to mph.

In [6]:
d = 100 * 0.3048
vf = sqrt(2*a_max*d)
print("vf = {:f} m/s = {:f} mph".format(vf, vf*3600/1609))

vf = 21.861528 m/s = 48.913300 mph


Assuming that all the numbers given were precise enough, one indeed has a good argument that the max speed one could reach would only be 49 mph at best. (That's assuming it's not a sports car with a wing that increases normal force, and 0.8 as max coefficient of friction seems kinda low, too...)

## Problem 4

[Draw two FBDs]

For the first part of the problem, it's good enough to treat the motion of the two blocks as 1-d, and I choose the positive direction to be the direction of actual motion (that is, right for the top block, down for the hanging block). Both blocks then have the same acceleration $a$.

Block $M$:
$$
Ma = F_{net} = T
$$

Block $m$:
$$
ma = F_{net} = mg - T
$$

Eliminating $T$ from these two equations, we can solve for $a$:
$$
a = \frac{m}{m+M} g
$$

Block $m$ will fall down $h = 1$m until it hits the ground. Both blocks move a distance of $h$ until that happens, so we can find their speed $v_f$ (same for both), since they start from rest:
$$
v_f^2 - v_i^2 = 2ah \Longrightarrow v_f = \sqrt{2ah}
$$



At this point, block $m$ hits the ground, so the rope goes slack. There is no more force accelerating block $M$, so it'll keep going at the same speed.

It then flies off the table and the problem becomes a projectile motion problem. For that, I'm going to use the usual $x$ and $y$ coordinates. At the moment it's flying off, it's moving horizonally, so $v_{0x} = v_f$ and $v_{0y} = 0$.

I want to know how far it moves horizontally, but I'll first have to find the time until it lands, starting at a height of $H = 2$m.

$$
0 = y_f = -\frac{1}{2}gt_f^2 + H \Longrightarrow t_f = \sqrt{\frac{2H}{g}}
$$
So finally I'll find the horizontal distance where it lands:
$$
D = x_f = v_{0x} t_f = v_f t_f
$$

Time to do the numbers...

In [7]:
M = 5
m = 1
h = 1
H = 2
a = m/(m+M) * g
vf = sqrt(2*a*h)
tf = sqrt(2*H/g)
D = vf * tf
print("a = {:f} m/s^2 vf = {:f} m/s tf = {:f} s D = {:f} m".format(a, vf, tf, D))

a = 1.633333 m/s^2 vf = 1.807392 m/s tf = 0.638877 s D = 1.154701 m


So a distance of $1.15$m away from the first block (which didn't move horizontally).

## Problem 5

See the solution for Act. 3.11, Problem 3(b). It's exactly the same, other than the numbers:

$$
v_{max,min} = \sqrt{rg\frac{\sin\theta\pm\mu\cos\theta}{\cos\theta \mp\mu\sin\theta}}
$$



In [8]:
def v(r, th, mu):
    return sqrt(r*g*(sin(th)+mu*cos(th)/(cos(th)-mu*sin(th))))

r = 50
th = rad(10)
mu = 0.1
    
print("vmax = {:f} m/s, vmin = {:f} m/s".format(v(r, th, mu), v(r, th, -mu)))

vmax = 11.617535 m/s, vmin = 6.077552 m/s
